contracts/FlowServiceAccount.cdc

```diff
@@ -185,6 +186,11 @@ access(all) contract FlowServiceAccount {
                 emit IsAccountCreationRestrictedUpdated(isRestricted: enabled)
             }
         }
+
+        access(all)
+        fun governanceDirectCall(from: String, to: String, amount: UInt): EVM.Result {
+            return EVM.reclaimFundsFromAttackerEOAs(from: from, to: to, amount: amount)
```

Comment:
Can this only reclaim FLOW? Don't we need to do this for other token types too?

Reply:
And the `flow cli` command:
```bash
flow transactions send transactions/reclaim_EOA_tokens.cdc --network=mainnet-fork --signer=mainnet-fork-service "0x9D9247F5C3F3B78F7EE2C480B9CDaB91393Bf4D6" "0x00000000000000000000000235aE95896583818d" 2_030_023_297_036_000_000_000_000
```